other-docs/guides/upgrading/v5.md

```diff
@@ -53,6 +55,14 @@ If you're using Facebook Instant Articles, you will need to require the FBIA plu
 $ composer require wpackagist-plugin/fb-instant-articles
 ```
 
+#### Sitemaps
+
+The MSM Sitemaps plugin used to provide the XML Sitemaps feature however this has been superceded by the new core Sitemaps in WordPress 5.5. These were a collaboration between WordPress, Google and Human Made.
+
+The update may require you to re-submit your sitemap index file in [Google Search Console](https://search.google.com/search-console/) as it has been relocated from `/sitemap.xml` to `/wp-sitemap.xml`. The new index sitemap URL is automatically added to the `robots.txt` file.
```

Comment:
Will the old sitemap URL definitely be no longer present? Do they need to keep it around in `robots.txt` if they had done any customization previously?

Reply:
Correct, if anyone was using the built in MSM one it was being added to `robots.txt` in the same way and was also generating it so it's a clean switch in that regard.

If there have been any customisations then those will need to be ported across.